azext_edge/edge/util/az_client.py

```diff
@@ -197,6 +197,7 @@ def wait_for_terminal_state(poller: "LROPoller", wait_sec: int = POLL_WAIT_SEC)
 def wait_for_terminal_states(
     *pollers: "LROPoller", retries: int = POLL_RETRIES, wait_sec: int = POLL_WAIT_SEC, **_
 ) -> Tuple["LROPoller"]:
+    from time import sleep
```

Comment:
There is a different pattern on how to achieve this in other resource areas, would be nice to be consistent

Reply:
we are trying to cut down on the time for the unit tests to run since it starts out with sleep (15) right away and that makes any test using that (asset/endpoint) extra slow

when you say pattern - do you mean:
1. push the sleep import back where it was (outside of functions)
2. change the patched thing in mocked_responses to mocker.patch(f"{AZ_CLIENT_PATH}.sleep")?

Reply:
1) Yes
2) Look at how create/update wait is configured in other resources that implement it. Examples:
- https://github.com/Azure/azure-iot-ops-cli-extension/blob/feature/0.7.0/azext_edge/edge/commands_edge.py#L258,
- https://github.com/Azure/azure-iot-ops-cli-extension/blob/feature/0.7.0/azext_edge/tests/edge/orchestration/resources/test_instances_unit.py#L189